schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+-- schemas
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(500)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount VARCHAR(100),
```

Comment:
- [ ] Great job! The major need for change here is the `data-type` of your data. The requirement specifies `decimal` but you seem to have `varchar`.

![image](https://github.com/wacoo/clinic-database/assets/99715640/10f64b7b-9d7c-4d96-981f-512fc35e6567)

- A good suggestion would be to ensure that `total_amount` is defined as `DECIMAL(10,2)`, which is a numeric data type with 10 digits in total and 2 decimal places. This is a more suitable choice for storing monetary values as it allows for mathematical calculations.